### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by mrpostiga** [OmniSharp/omnisharp-atom#70]
atom.services is no longer available. To register service providers and
consumers, use the `providedServices` and `consumedServices` fields in
your package's package.json.
```
Atom.Object.defineProperty.get (C:\Users\mrpostiga\AppData\Local\atom\app-0.187.0\resources\app\src\atom.js:64:11)
Completion.registerProviders (C:\Users\mrpostiga\.atom\packages\omnisharp-atom\lib\omnisharp-atom\features\completion.coffee:11:32)
```

**Comment by ctolkien** [comment]
This is addressed in #64

**Comment by ctolkien** [comment]
Fixed.
